core/shared/src/main/scala/org/http4s/Http4s.scala

```diff
@@ -0,0 +1,35 @@
+/*
+ * Copyright 2013 http4s.org
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.http4s
+
+@deprecated("Use org.http4s.implicits._ instead", "0.20.0-M2")
+trait Http4s extends Http4sInstances with Http4sFunctions with syntax.AllSyntax
```

Comment:
I think we removed this entire file in 0.23 and it can be dropped here.

Reply:
Cool! Gone!